router.php

```diff
@@ -1,18 +1,21 @@
 <?php
 // Used by `wp server` to route requests.
 
-namespace WP_CLI\Router;
+namespace WP_CLI\Server\Router;
 
 /**
  * This is a copy of WordPress's add_filter() function.
  *
  * We duplicate it because WordPress is not loaded yet.
  */
-function add_filter($tag, $function_to_add, $priority = 10, $accepted_args = 1) {
+function add_filter( $tag, $function_to_add, $priority = 10, $accepted_args = 1 ) {
 	global $wp_filter, $merged_filters;
 
-	$idx = _wp_filter_build_unique_id($tag, $function_to_add, $priority);
-	$wp_filter[$tag][$priority][$idx] = array('function' => $function_to_add, 'accepted_args' => $accepted_args);
+	$idx                                    = _wp_filter_build_unique_id( $tag, $function_to_add, $priority );
```

Comment:
Let's add a newline here to avoid the extreme alignment that pushes the code too much to the right.
```suggestion
	$idx = _wp_filter_build_unique_id( $tag, $function_to_add, $priority );

```